### PR TITLE
New exercise and edits to slides

### PR DESCRIPTION
**Pull request by brendajin**
@cdersky I ended up adding the new credit card assignment – let me know what you think!

### CODE REVIEW

**Comment by cdersky** [comment]
Please update the Topics Covered section in the meetup announcement to clarify which topics are covered which weekend/part. (The first two bullet points are covered the first weekend, the last two the last weekend)

**Comment by brendajin** [comment]
@cdersky made some changes. does that look good now? or was I supposed to change the `javascript/index.html` teaching guide?

File: javascript/exercises/creditcard.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+<p>The following credit card numbers are valid:</p>
+<pre><code>9999-9999-8888-0000
+6666-6666-6666-1666
+</code></pre>
+
+<p>The following credit card numbers are invalid:
+  <ul>
+    <li><code>a923-3211-9c01-1112</code> <em>invalid characters</em></li>
+    <li><code>4444-4444-4444-4444</code> <em>only one type of number</em></li>
+    <li><code>1211-1111-1111-1112</code> <em>sum less than 16</em></li>
```
Comment (cdersky) [inline javascript/exercises/creditcard.html]: the sum here is 18. Either change the rule or change the example to 1111-1111-1111-1111.

**Comment by cdersky** [comment]
@brendajin, once the minor change to the CC exercise example is made I will give a thumbs up.

**Comment by brendajin** [comment]
@cdersky fixed the homework issue. how does it look now?

**Comment by cdersky** [comment]
@brendajin All of the pages display nicely by the browser. Just two notes:
* The image in javascript/images/console.png is outdated. The validateCreditCard function should take in a string of numbers with no dashes.
* The creditCard exercise will updated in the next PR

**Pull request merged**